packages/extension/src/contexts/LedgerContext/LedgerContext.tsx

```diff
@@ -75,15 +87,15 @@ const LedgerProvider: FC = ({ children }) => {
         // Prepare the transaction
         const prepared = await client.request({
           command: 'account_nfts',
-          account: wallet.publicAddress,
+          account: 'rDGh681kc6V1GKkQB378XhiM1tzkYrnFwQ', // Change before merging to master
           limit: payload?.limit,
           marker: payload?.marker,
           ledger_index: 'validated'
         });
         if (!prepared.result?.account_nfts) {
           throw new Error("Couldn't get the NFTs");
         } else {
-          return prepared.result.account_nfts;
+          return { account_nfts: prepared.result.account_nfts, marker: prepared.result.marker };
```

Comment:
This issue has been addressed in an earlier MR

Reply:
I will have a look at it, I think the previous change is changing the API, this would create a breaking change, we cannot update it as is without an API release, let me check this in your other MR.